src/commands/pardon.js

```diff
@@ -0,0 +1,284 @@
+const Discord = require('discord.js');
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const Warnings = require('../models/warnings');
+const Kicks = require('../models/kicks');
+const Bans = require('../models/bans');
+const util = require('../util');
+const warn = require('./warn');
+
+/**
+ *
+ * @param {Discord.CommandInteraction} interaction
+ */
+async function pardonHandler(interaction) {
+	await interaction.deferReply({
+		ephemeral: true
+	});
+
+	const guild = await interaction.guild.fetch();
+	const executingMember = await interaction.member.fetch();
+	const executor = executingMember.user;
+	const type = interaction.options.getString('type');
+	const users = interaction.options.getString('users');
+	const reason = interaction.options.getString('reason');
+
+	const userIds = [...new Set(Array.from(users.matchAll(Discord.MessageMentions.USERS_PATTERN), match => match[1]))];
+
+	const image = new Discord.MessageAttachment('./src/images/mod/mod-pardon.png');
+	// Initialize all of the embeds
+	const pardonedListEmbed = new Discord.MessageEmbed();
+	const eventLogEmbed = new Discord.MessageEmbed();
+	const pardonDmEmbed = new Discord.MessageEmbed();
+
+	for (const userId of userIds) { 
+		const member = await interaction.guild.members.fetch(userId);
+		const user = member.user;
+
+		if (type === 'warn_type') { // If chosen warn within options
```

Comment:
Mostly a nitpick. This goes for all comments, but we prefer for comments to be on their own lines rather in situations like this, rather than besides a brace